test/functional/test_framework/test_node.py

```diff
@@ -212,8 +212,12 @@ def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
 
     def assert_start_raises_init_error(self, extra_args=None, expected_msg=None, *args, **kwargs):
         """Attempt to start the node and expect it to raise an error.
+
+        extra_args: extra arguments to pass through to bitcoind
+        expected_msg: regex that stderr should match when bitcoind fails
```

Comment:
-> pivxd

Reply:
Added e6119ed182c1b658fb949d9efaee6fb8b31932eb to update all comments in the test suite